build/images/scripts/start_ovs

```diff
@@ -50,7 +54,12 @@ function quit {
 # exit with code 128 + SIGNAL
 trap "quit" INT TERM
 
-start_ovs
+offload=false
+if [ -z "$1" ] && [ $1 == "hw-offload" ];then
```

Comment:
could we use `--hw-offload` to be consistent with another flag that we have for a similar script (`start_ovs_netdev`)?